komodod/confd/templates/komodo.conf.tmpl

```diff
@@ -2,6 +2,7 @@ rpcuser={{ or (getenv "KOMODO_RPC_USER") "rpcuser" }}
 rpcpassword={{ or (getenv "KOMODO_RPC_PASSWORD") "rpcpassword" }}
 server=1
 txindex=1
+blocknotify=/home/komodo/ScaleTestV2/blocknotify.sh %s
```

Comment:
If we keep this line here, both AC and Miner containers will execute this script. If this is not result what we want, we need to pass it as `-blocknotify=...` from docker-compose.yml file.

Reply:
This actually does nothing, it's only used as komodo.conf and didnt actually work for assets. I thought I already removed it lol.